### PR TITLE
Stylelint updates

### PR DESCRIPTION
**Pull request by alanmoo**
Update stylelint dependencies and .scss files to match. Just code cleanup.

@gvn or @mmmavis this should be a < 2 minute review if you have a moment.

### CODE REVIEW

File: app/react/components/data-card/data-card.scss [hunk 1]
```diff
@@ -11,16 +11,19 @@ $cardPadding: 1.25rem;
...
   font-weight: 300;
   width: 100%;
+
   .card-image {
-    margin: 0 -1*$cardPadding;
+    margin: 0 (-1 * $cardPadding);
     width: calc(100% + #{$cardPadding*2});
     max-width: inherit;
-    transform: translateY(-1*$cardPadding);
+    transform: translateY(-1 * $cardPadding);
```
Comment (alanmoo) [inline app/react/components/data-card/data-card.scss]: Yes, sass calculates this properly still.

**Comment by alanmoo** [comment]
Scratch that, forgot I updated react and noticed a change that shouldn't be there. Still working on this.

**Comment by alanmoo** [comment]
Ready now

**Comment by mmmavis** [comment]
R+ 👍

**Pull request merged**